Sources/TuistKit/Services/GraphImplicitImportLintService.swift

```diff
@@ -0,0 +1,50 @@
+import Path
+import Tools
+import TuistCore
+import TuistGenerator
+import TuistLoader
+import XcodeGraph
+
+final class GraphImplicitImportLintService {
```

Comment:
Should this conform to the `GraphLinting` protocol? Otherwise, we can probably remove the Config from the function signature [here](https://github.com/tuist/tuist/pull/6516/files#diff-432d00a701dd55696c8c99219c1ea800d4fe63626e0a6a1d36d93e55f753dbdaR15)